torch/nn/quantized/modules/conv.py

```diff
@@ -12,13 +12,15 @@
 import torch.nn.intrinsic.qat as nniqat
 
 from torch._ops import ops
-from torch.nn.modules.utils import _pair, _triple
+from torch.nn.modules.utils import _single, _pair, _triple
 from torch.nn.quantized.modules.utils import _quantize_weight
 from torch.nn.utils import fuse_conv_bn_weights
 
 class _ConvNd(nn.Module):
     def __init__(self, in_channels, out_channels, kernel_size, stride=1,
-                 padding=0, dilation=1, groups=1, bias=True,
+                 padding=0, dilation=1,
+                 transposed=False, output_padding=0,
+                 groups=1, bias=True,
```

Comment:
can we align with `_ConvNd` in `nn`? https://github.com/pytorch/pytorch/blob/master/torch/nn/modules/conv.py#L20-L22, the default arguments seems to be put in the actual Conv modules like `Conv1d`, `Conv2d` etc.